geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterConfigurationService.java

```diff
@@ -352,6 +358,20 @@ public boolean removeJars(final String[] jarNames, String[] groups) {
         if (configuration == null) {
           break;
         }
+
+        for (String jarRemoved : jarNames) {
+          File jar = this.getPathToJarOnThisLocator(group, jarRemoved).toFile();
+          if (jar.exists()) {
+            try {
+              FileUtils.forceDelete(jar);
+            } catch (IOException e) {
+              logger.error(
+                  "Exception occurred while attempting to delete a jar from the filesystem: {}",
+                  jarRemoved, e);
+            }
```

Comment:
Is it safe for this exception to be swallowed?  I'll admit I'm shaky on how the classloader works, but is there a risk that the jar could still get picked up if it remains on the classpath?  Or does the cluster config remove that worry?